tools/mo/openvino/tools/mo/utils/versions_checker.py

```diff
@@ -287,9 +285,17 @@ def check_requirements(framework=None):
             continue
 
     if len(not_satisfied_versions) != 0:
-        extension = 'bat' if os.name == 'nt' else 'sh'
-        install_file = 'install_prerequisites{0}.{1}'.format(framework_suffix, extension)
```

Comment:
We removing hint to use install_prerequisites if some dependencies aren't satisfied, instead we ask to use pip install openvino-dev, but installation through pip is not the only way to install the openvino, are we sure this is correct to ask to use pip?

Reply:
Currently we have only pip. Other sources of distributions like yum, apt, pset were removed. Regular installer installs pip packages as well.

Reply:
I have clarified this with @slyubimt